.travis.yml

```diff
@@ -7,6 +7,7 @@ env:
 before_script:
   - npm run db:setup:user
   - npm run db:reset
+  - npm run db:seed
```

Comment:
Did you find that adding this was necessary? Usually your tests should depend on factories that you add into the DB within the test lifecycle as opposed to relying on seed data that we insert before all of the tests. Happy to talk about this more IRL.